tcti/tcti-mssim.c

```diff
@@ -508,13 +508,13 @@ Tss2_Tcti_Mssim_Init (
     char hostname[HOST_NAME_MAX + 1] = { 0 };
     uint16_t port = TCTI_SOCKET_DEFAULT_PORT;
 
+    LOG_TRACE ("tctiContext: 0x%" PRIxPTR ", size: 0x%" PRIxPTR ", conf: %s",
+               (uintptr_t)tctiContext, (uintptr_t)size, uri_str);
     if (tctiContext == NULL && size == NULL) {
         return TSS2_TCTI_RC_BAD_VALUE;
-    } else if( tctiContext == NULL ) {
+    } else if (tctiContext == NULL) {
```

Comment:
Didn't mean for this formatting fix to sneak in here. It's a fix we'd want but if you want me to refactor this into another commit just say the word.

Reply:
No big deal.